packages/vuetify/src/components/VNavigationDrawer/VNavigationDrawer.tsx

```diff
@@ -39,10 +40,14 @@ export default defineComponent({
       type: [Number, String],
       default: 256,
     },
+    position: {
+      type: String as PropType<'left' | 'right' | 'bottom'>,
+      default: 'left',
+      validator: (value: any) => ['left', 'right', 'bottom'].includes(value),
```

Comment:
is `'top'` omitted here purposely? does adding it break anything or just seems very unlikely to be used?

Reply:
bottom had a class associated with it, but top didn't, so I assumed it wasn't supported.

Reply:
There is no support for **top** in `v-navigation-drawer`.